.github/workflows/update-esy-dependencies.yml

```diff
@@ -0,0 +1,54 @@
+on:
+  schedule:
+    - cron: "0 0 * * 0"
```

Comment:
@anmonteiro I wasn't sure what schedule would be good - feel free to change this however you'd like.